kubernetes/resource_kubernetes_secret_test.go

```diff
@@ -165,7 +168,7 @@ func TestAccKubernetesSecret_importBasic(t *testing.T) {
 		CheckDestroy: testAccCheckKubernetesSecretDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccKubernetesSecretConfig_basic(name),
+				Config: testAccKubernetesSecretConfig_noBase64(name),
```

Comment:
The `TestAccKubernetesSecret_importBasic` should use the `testAccKubernetesSecretConfig_basic` configuration.

Reply:
thanks. done.